pyp2rpm/package_getters.py

```diff
@@ -127,25 +127,28 @@ def __init__(self, client, name, version=None, prerelease=False,
                  save_dir=None):
         self.client = client
         self.name = name
-        self.versions = self.client.package_releases(self.name, True)
-
-        # Use only stable versions, unless --pre was specified
-        if not prerelease:
-            self.versions = [candidate for candidate in self.versions
-                             if not parse_version(candidate).is_prerelease]
+        if version:
+            # if version is specified, will check if such version exists
+            if self.client.release_urls(name, version) == []:
```

Comment:
I think `if not self.client.release_urls(…):` is more readable (as empty lists are False-y).